---a/net/minecraft/block/BlockObsidian.java
+++b/net/minecraft/block/BlockObsidian.java
@@ -1,6 +1,7 @@
 package net.minecraft.block;
 
 import java.util.Random;
+
 import net.minecraft.block.material.MapColor;
 import net.minecraft.block.material.Material;
 import net.minecraft.block.state.IBlockState;
@@ -8,24 +8,21 @@
 import net.minecraft.init.Blocks;
 import net.minecraft.item.Item;
 
-public class BlockObsidian extends Block
-{
+public class BlockObsidian extends Block {
+
     private static final String __OBFID = "CL_00000279";
 
-    public BlockObsidian()
-    {
+    public BlockObsidian() {
         super(Material.rock);
         this.setCreativeTab(CreativeTabs.tabBlock);
     }
 
-    public Item getItemDropped(IBlockState state, Random rand, int fortune)
-    {
+    public Item getItemDropped(IBlockState state, Random rand, int fortune) {
         return Item.getItemFromBlock(Blocks.obsidian);
     }
 
-    public MapColor getMapColor(IBlockState state)
-    {
+    public MapColor getMapColor(IBlockState state) {
         return MapColor.obsidianColor;
     }
-}
+}
